aeron-driver/src/main/c/aeronmd.h

```diff
@@ -738,25 +738,6 @@ int aeron_driver_close(aeron_driver_t *driver);
  */
 int aeron_delete_directory(const char *dirname);
 
-/**
- * Clock function used by aeron.
- */
-typedef int64_t (*aeron_clock_func_t)();
-
-/**
- * Return time in nanoseconds for machine. Is not wall clock time.
- *
- * @return nanoseconds since epoch for machine.
- */
-int64_t aeron_nano_clock();
```

Comment:
These are part of the API and thus should not be removed from the header.

Reply:
aeronmd.h is the single header file for the API. So, the functions should be available with just this header file.

Reply:
There's a couple of places where being able to include the aeron_clock.h instead of aeronmd.h is useful.  Would splitting aeron_clock.h into aeron_clock_api.h (for aeron_nano_clock and aeron_epoch_clock) and aeron_clock.h (all the rest) and have aeronmd.h and aeron_clock.h include aeron_clock_api.h work?

Reply:
Does the cached clock need to become part of the API?

Reply:
I don't think the cached clock needs to be part of the API. If needed, then it could be included via pulling in the other headers. And also, the cached clock needs to be updated, etc. While the ms and ns clocks are simply single functions.

In terms of the header, I would like to keep the single include file if possible.